content/docs/quick-start/authentication.mdx

```diff
@@ -9,44 +8,32 @@ the request you're making in the header of request you make.
 Generally, we provide both public and secret keys. Public keys are meant to be used from your front-
 end when integrating Lazerpay Inline Checkout and our Frontend libraries. By design, public keys cannot modify any part of your account besides initiating transactions. The secret keys however, are to be kept secret. If for any reason you believe your secret key has been compromised or you wish to reset them, you can do so from the dashboard.
 
-
-
-
     <Alert 
     status="warning" 
     title="Secure your requests"
     className="my-2" 
     content=' Do not commit your secret keys to Git, or use them in client-side code..'/>
 
-<Divider className="max-w-100 mb-10  mt-5" />
-
+<Divider className='max-w-100  mb-12' />
```

Comment:
Please check if `mb-12` works here.  Still doesn't match the design

Reply:
![image](https://user-images.githubusercontent.com/52764879/177113074-798d3697-5242-4d6e-b859-9e8291dd686a.jpeg)

Hi this is what it looks like with mb-12

Reply:
I mean, it's currently `mb-12`, which i don't think works